neon/data-collection/scene-camera-exposure/index.md

```diff
@@ -0,0 +1,44 @@
+# Scene Camera Exposure
+The [scene camera’s](https://docs.pupil-labs.com/neon/data-collection/data-streams/#scene-video) exposure modes can be adjusted to improve image quality in different lighting conditions. There are four modes:
+
+- **Manual:** This mode lets you set the exposure value manually.
+- **Automatic**: `Highlights`, `Balanced`, and `Shadows` modes automatically adjust exposure according to the surrounding lighting.
+    - `Highlights` mode optimizes the exposure to capture bright areas in the environment, while potentially underexposing dark areas.
+    - `Balanced` mode optimizes the exposure to capture brighter and darker areas in the environment equally.
+    - `Shadows` mode optimizes the exposure to capture darker areas in the environment, while potentially overexposing bright areas.
```

Comment:
All of this information is duplicated below, by the screenshots. We should keep one or the other. I suggest we keep the screenshots + text description.